code/style.css

```diff
@@ -0,0 +1,183 @@
+body {
+  background: #fff;
+  color: white;
+  margin: 0;
+  font-family: 'Lato', Sans-Serif;
+}
+
+h1 {
+  margin: 0;
+}
+
+h2 {
+  font-size: 32px;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+}
+
+/****** HEADER ******/
+
+header {
+  position: relative;
+  color: #000;
+  display: flex;
+  font-family:'Arvo', Serif;
+  align-items: center;
+  justify-content: center;
+  height: 600px;
+}
+
+.header-title {
+  font-size: 72px;
+  text-align: center;
+}
+
+video {
+  display: block;
+  position: absolute;
+  z-index: -1;
+  margin: 0;
+  right: 0;
+  top: 0;
+  min-width: 100%;
+  height: 600px;
+  object-fit: cover;
+}
+
+/****** MAIN ******/
+
+main {
```

Comment:
that's a lot of styling by selector - in ideal situation our CSS file should consist only from class styling